src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -126,6 +126,10 @@ class AktivitetskravService(
         aktivitetskravRepository.getAktivitetskrav(personIdent = personIdent, connection = connection)
             .map { it.toAktivitetskrav() }
 
+    fun getAktivitetskravForPersons(personidenter: List<PersonIdent>): Map<PersonIdent, Aktivitetskrav> =
+        aktivitetskravRepository.getAktivitetskravForPersons(personidenter = personidenter)
+            .map { it.personIdent to it.mapMostRecentVurdering() }.toMap()
```

Comment:
IntelliJ mener man kan bruke `associate()` her i stedet for først `.map` og så `toMap()`.